jina/executors/indexers/vector.py

```diff
@@ -2,33 +2,17 @@
 __license__ = "Apache-2.0"
 
 import gzip
-import os
-from functools import lru_cache
 from os import path
-from typing import Optional, List, Union, Tuple, Dict
+from typing import Optional, List, Union, Tuple
 
 import numpy as np
 
 from . import BaseVectorIndexer
-from ..decorators import batching
 from ...helper import cached_property
 
 
 class BaseNumpyIndexer(BaseVectorIndexer):
```

Comment:
this file has been changed in a recent merge from master and the update has not been properly done!